### ISSUE TITLE
Questions about hierarchical quality in DCVC-RT

### ISSUE THREAD

**Issue by BridgesK** [microsoft/DCVC#91]
Thank you for your great work on DCVC-RT. I have a question regarding the usage of qp offset and the corresponding weight setting during training.

In DCVC-RT, it is mentioned that "the qp offset is set to **[0, 8, 0, 4, 0, 4, 0, 4]** for hierarchical quality." My understanding is that one qp offset corresponds to one $w_t$ for the distortion term in the loss function. Therefore, it seems that **8 consecutive frames** would be required during training. However, the Vimeo-90k dataset provides only **7** frames per video, which leaves me a bit confused about how the qp offset is applied in this case.

Additionally, I noticed that in DCVC-DC, the length of the weight setting is **4** (i.e., 0.5, 1.2, 0.5, 0.9). Could you please clarify:

- How is qp offset utilized on Vimeo-90k given the discrepancy in the number of frames?

- What is the relationship between qp offset and the $w_t$ in the training process?

Thank you very much !

**Comment by yaohualibin** [comment]
As described in DCVC-RT paper, the we processed the original Vimeo videos [5] to create longer sequences for fine-tuning by following [28].
Roughly, 0 ==> 0.5, 8 ==> 1.2, and 4 ==> 0.9, mapping qp offset with the weight having the same index.

**Comment by BridgesK** [comment]
> As described in DCVC-RT paper, the we processed the original Vimeo videos [5] to create longer sequences for fine-tuning by following [28]. Roughly, 0 ==> 0.5, 8 ==> 1.2, and 4 ==> 0.9, mapping qp offset with the weight having the same index.

Thank you for your explanation. So during training (with Vimeo-90k’s 7-frame sequences), the qp offset is set to [0, 8, 0, 4]?
